shared/src/androidMain/kotlin/main.android.kt

```diff
@@ -21,29 +20,28 @@ import at.asitplus.wallet.app.common.dcapi.CredentialsContainer
 import at.asitplus.wallet.app.common.dcapi.DCAPIRequest
 import at.asitplus.wallet.app.common.dcapi.ResponseJSON
 import com.android.identity.android.mdoc.util.CredmanUtil
-import com.android.identity.crypto.Algorithm
-import com.android.identity.crypto.Crypto
-import com.android.identity.crypto.EcCurve
-import com.android.identity.crypto.EcPublicKeyDoubleCoordinate
+import org.multipaz.crypto.Algorithm
```

Comment:
Please reorder imports automatically, so it won't show up as a change in the next edit of this file